### PR TITLE
 "[reland] [reland] add ability to view multiple focused log lines at on… (#4659)"

### PR DESCRIPTION
**Pull request by PaliC**
"[reland] [reland] add ability to view multiple focused log lines at on… (#4659)"

Original commit changeset: 06978f08a149

I'm reasonably confident this works now, though can someone confirm through checking on Vercel

---
Stack created with [Sapling](https://sapling-scm.com). Best reviewed with [ReviewStack](https://reviewstack.dev/pytorch/test-infra/pull/4660).
* #4640
* #4639
* __->__ #4660

### CODE REVIEW

File: torchci/components/CommitStatus.tsx [hunk 1]
```diff
@@ -92,6 +92,7 @@ export default function CommitStatus({
           !isRerunDisabledTestsJob(job) &&
           !isUnstableJob(job)
         }
+        // pred={(job) => job.conclusion === "pending"}
```
Comment (huydhn) [inline torchci/components/CommitStatus.tsx:95]: Commented out?

File: torchci/components/LogViewer.tsx [hunk 2]
```diff
@@ -194,13 +194,22 @@ function Log({ url, line }: { url: string; line: number | null }) {
...
+  showAnnotationToggle = process.env.DEBUG_LOG_CLASSIFIER === "true",
+  maxNumFailureLines = process.env.DEBUG_LOG_CLASSIFIER === "true" ? 2 : 1,
 }: {
   job: JobData;
   logRating?: LogAnnotation;
   showAnnotationToggle?: boolean;
+  maxNumFailureLines?: number;
 }) {
-  const [showLogViewer, setShowLogViewer] = useState(false);
+  // const numFailureLines =
```
Comment (huydhn) [inline torchci/components/LogViewer.tsx:205]: Same here, are these lines commented out for debugging?

**Comment by huydhn** [comment]
On this preview page https://torchci-git-pr4660-fbopensource.vercel.app/pytorch/pytorch/commit/18cc8a92aca8529b9c689b1c64f70bf86509e258, the last job does have a log viewer.  Does that look correct to you?

File: torchci/components/CommitStatus.tsx (hunk 1 above)
Comment (PaliC) [reply torchci/components/CommitStatus.tsx:95]: Thanks for the catch, yeah it's for debugging

File: torchci/components/LogViewer.tsx (hunk 2 above)
Comment (PaliC) [reply torchci/components/LogViewer.tsx:205]: This one specifically is not.

**Pull request merged**